hr_hr.lang

```diff
@@ -1,215 +1,349 @@
+gui.xaero_display_waypoints=Putne točke na mapi
+gui.xaero_display_players=Prikaži igrače
+gui.xaero_display_mobs=Prikaži živa bića
+gui.xaero_display_items=Prikaži predmete
+gui.xaero_display_other=Prikaži ostalo
+gui.xaero_zoom=Zoom
+gui.xaero_minimap_size=Veličina mini karte
+gui.xaero_entity_amount=Količina entiteta
+gui.xaero_black=Crna
+gui.xaero_dark_blue=Tamno plava
+gui.xaero_dark_green=Tamno zelena
+gui.xaero_dark_aqua=Tamno tirkizna
+gui.xaero_dark_red=Tamno crvena
+gui.xaero_dark_purple=Tamno ljubičasta
+gui.xaero_gold=Zlatna
+gui.xaero_gray=Siva
+gui.xaero_dark_gray=Tamno siva
+gui.xaero_blue=Plava
+gui.xaero_green=Zelena
+gui.xaero_aqua=Tirkizna
+gui.xaero_red=Crvena
+gui.xaero_purple=Ljubičasta
+gui.xaero_yellow=Žuta
+gui.xaero_white=Bijela
+gui.xaero_percent=Postotak
+gui.xaero_numeral=Broj
+gui.xaero_both=Oboje
+gui.xaero_tiny=Sićušno
+gui.xaero_small=Malo
+gui.xaero_medium=Srednje
+gui.xaero_large=Veliko
+gui.xaero_pvp_sprint=PVP trčanje
+gui.xaero_quick_eat=Brzo jedenje
+gui.xaero_toggle_sneak=Prebaci šuljanje
+gui.xaero_quick_use_1=Brzo korištenje 1
+gui.xaero_quick_use_2=Brzo korištenje 2
+gui.xaero_quick_use_3=Brzo korištenje 3
+gui.xaero_quick_use_4=Brzo korištenje 4
+gui.xaero_quick_use_5=Brzo korištenje 5
+gui.xaero_quick_use_6=Brzo korištenje 6
+gui.xaero_quick_use_7=Brzo korištenje 7
+gui.xaero_quick_use_8=Brzo korištenje 8
+gui.xaero_quick_use_9=Brzo korištenje 9
+gui.xaero_zoom_in=Zumiranje mini karte
+gui.xaero_zoom_out=Odzumiranje mini karte
+gui.xaero_on=Da
+gui.xaero_off=Ne
+gui.xaero_restores=Obnavlja
+gui.xaero_preset_topleft=Gore-lijevo
+gui.xaero_preset_topright=Gore-desno
+gui.xaero_waypoint_teleport=Teleportiraj
+gui.xaero_waypoints_key=Putne točke
+gui.xaero_display_coords=Prikaži koordinate
+gui.xaero_disable_enable=Omogući/Onemogući
+gui.xaero_disabled=onemogućeno
+gui.xaero_minimap_guide=Klikni i povlači za pomicanje mini karte.
+gui.xaero_change_position=Promijeni poziciju
+gui.xaero_preset_bottom_left=Dolje-lijevo
+gui.xaero_preset_bottom_right=Dolje-desno
+gui.xaero_ingame_waypoints=Putne točke u igri
+gui.xaero_next=Sljedeće >>
+gui.xaero_previous=<< Prethodno
+gui.xaero_lock_north=Zaključaj sjever
+gui.xaero_deathpoint=Posljednja smrt
+gui.xaero_deathpoint_old=Stara smrt
+gui.xaero_deathpoints=Putne točke smrti
+gui.xaero_chunkgrid=Rešetka dijelova svijeta
+gui.xaero_entity_colours=Boje entiteta
+gui.xaero_players=Igrači
+gui.xaero_mobs=Prijateljska bića
+gui.xaero_hostile=Neprijateljska bića
+gui.xaero_items=Predmeti
+gui.xaero_other=Ostali entiteti
+gui.xaero_hunger_low=Niska glad
+gui.xaero_hp_low=Nisko zdravlje
+gui.xaero_enlarge_map=Povećaj mini kartu
+gui.xaero_fbo=Beta
+gui.xaero_potion_status_settings=Postavke stanja napit(a)ka
+gui.xaero_potion_names=Imena efekata napitaka
+gui.xaero_opacity=Neprozirnost
+gui.xaero_slime_chunks=Dijelovi svijeta s ljigavcima
+gui.xaero_unlimited=Neograničeno
+gui.xaero_old_deathpoints=Zadrži putne točke stare smrti
+gui.xaero_toggle_map=Uključi mini kartu

```

Comment:
Yes, it would work fine, I'll change it. Prebaci was taken from in game translation for toggle in sprint and sneak settings under accessibility. I added Uključi as I thought that Prebaci mini kartu could be confusing for people because it can mean 3 things. Literal translations are:
- Uključi - turn on
- Prebaci - move, toggle, switch

Reply:
I later decided not to put Prebaci, so text wasn't changed.

Reply:
Ok, got it. But isn't "turn on" incorrect? It also turns it off.
You can try something like "turn on/turn off", since there's a bit more space in the controls menu I think.